content/400-reference/300-database-reference/02-connection-urls.mdx

```diff
@@ -107,6 +107,8 @@ datasource db {
 
 You can then either set the environment variable in your terminal or by providing a [dotenv](https://github.com/motdotla/dotenv) file named `.env`. This will automatically be picked up by the Prisma CLI.
 
+The connection URL is read from the env-file during build time to generate the client as well as during run time to connect to the actual database.
```

Comment:
In which way is this relevant for Client during generate/build time?

Reply:
Yeah, but why do you need a `DATABASE_URL` value at all during `generate`/build?

Reply:
You have a point there. I have updated the documentation, by specifying that it is used while updating the prsima.schema when pulling the latest schema from the DB